tools/coqdep_common.ml

```diff
@@ -517,10 +396,8 @@ let rec suffixes = function
 
 let add_caml_known phys_dir _ f =
   let basename,suff =
-    get_extension f [".ml";".mli";".mlg";".mllib";".mlpack"] in
+    get_extension f [".mllib"; ".mlpack"] in
```

Comment:
Why does coqdep still have code for .mllib / .mlpack here?

Reply:
This is used to check existence of `Declare ML Plugin` deps.

What to do with it is a good question, but it doesn't hurt for now.